datafusion/sqllogictest/test_files/cast.slt

```diff
@@ -69,3 +69,16 @@ query ?
 SELECT CAST(MAKE_ARRAY() AS VARCHAR[])
 ----
 []
+
+statement ok
+create table t0(v0 BIGINT);
+
+statement ok
+insert into t0 values (1),(2),(3);
+
+query I
+select * from t0 where v0>1e100;
+----
+
```

Comment:
I suggest a test that also shows the filter passing (I wasn't sure if the whole thing was turned into NULL). But I double checked and this query works on this branch:

```suggestion
query I
select * from t0 where v0<1e100;
----
1
2
3

```

Reply:
I'll add that